core/tasks.py

```diff
@@ -220,6 +222,8 @@ def deferred(self, *args, **kwargs):
                                             retryCount))
         cmd, data = json.loads(req.body, object_hook=jsonDecodeObjectHook)
         funcPath, args, kwargs, env = data
+        logger.debug(f"Call task {funcPath} with cmd={cmd!r} args={args!r} kwargs={kwargs!r} env={env!r}")
```

Comment:
@ArneGudermann as @sveneberth suggested, please define this line as this, which is much shorter and a cool feature to use and spread.
```suggestion
        logger.debug(f"Call task {funcPath} with {cmd=} {args=} {kwargs=} {env=}")
```